File: VB/SpreadsheetControl_API_Part03/DisplayResultControl.vb
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraSpreadsheet

Namespace SpreadsheetControl_API_Part03

    Public Partial Class DisplayResultControl
        Inherits UserControl

        Public Sub New()
            InitializeComponent()
        End Sub

        Public ReadOnly Property Speadsheet As SpreadsheetControl
            Get
                Return spreadsheetControl1
            End Get
        End Property
    End Class
End Namespace

File: CS/SpreadsheetControl_API_Part03/DisplayResultControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraSpreadsheet;

namespace SpreadsheetControl_API_Part03
{
    public partial class DisplayResultControl : UserControl
    {
        public DisplayResultControl()
        {
            InitializeComponent();
        }

        public SpreadsheetControl Speadsheet { get { return spreadsheetControl1; } }

}
    }
